File: src/be-hunt-api/Hunts.Domain/Entities/Hunt.cs
namespace Hunts.Domain.Entities
{
    /// <summary>
    /// Represents a hunt with multiple assignments.
    /// </summary>
    public sealed class Hunt
    {
        /// <summary>
        /// Unique identifier of a hunt.
        /// </summary>
        public int Id { get; set; }

        /// <summary>
        /// A title usually contains a topic of a hunt.
        /// </summary>
        public string Title { get; set; } = default!;

        /// <summary>
        /// A description usually contains additional information of a hunt.
        /// </summary>
        public string Description { get; set; } = default!;

        /// <summary>
        /// Each hunt has multiple tasks to complete.
        /// </summary>
        public ICollection<Assignment> Assignments { get; set; } = [];
    }
}

File: old-files/src/fe-hunt-editor/src/lib/models/Hunt.ts
import type { Assignment } from './Assignment';

/**
 * Represents a hunt with multiple assignments.
 */
export interface Hunt {
	/** The unique identifier of the hunt. */
	id: number;
	/** The title of the hunt. */
	title: string;
	/** The description of the hunt. */
	description: string;
	/** The list of assignments in the hunt. */
	assignments: Assignment[];
}
